src/Components/ListOfGoods/ListOfGoods.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import { Goods } from '../../types';
+
+interface Props{
+  goods:Goods
+}
+export const ListOfGods: FC<Props> = (props) => {
```

Comment:
only for future, use naming that not related to name of data. For reusing this component